frontend/app/services/lookup-service.server.ts

```diff
@@ -376,6 +376,19 @@ function createLookupService() {
     return federalSocialPrograms;
   }
 
+  function getFederalSocialProgramsById(id: string) {
+    log.debug('Fetching federal social program with id: [%j]', id);
```

Comment:
Use `%s` instead of `%j` since `id` is not a JSON/Object type.

Same for `getProvincialTerritorialSocialProgramById`

https://github.com/winstonjs/winston?tab=readme-ov-file#string-interpolation

https://nodejs.org/dist/latest/docs/api/util.html#utilformatformat-args
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/37d4a484-6526-4474-9e60-737b0acf3854)